examples/simple-pipeline/simple_pipeline/pipeline.py

```diff
@@ -0,0 +1,100 @@
+"""Simple Pipeline Configuration.
+
+Authors:
+    Hermes Vincentius Gani (hermes.v.gani@gdplabs.id)
+    Irvan Ariyanto (irvan.ariyanto@gdplabs.id)
+
+References:
+    None
+"""
+
+from typing import Any
+
+from gllm_generation.response_synthesizer import StaticListResponseSynthesizer
+from gllm_pipeline.pipeline.pipeline import Pipeline
+from gllm_pipeline.steps import step
+from gllm_plugin.pipeline.pipeline_plugin import PipelineBuilderPlugin
+
+from simple_pipeline.preset_config import SimplePresetConfig
+from simple_pipeline.state import SimpleState, SimpleStateKeys
+
+
+class SimplePipelineBuilder(PipelineBuilderPlugin[SimpleState, SimplePresetConfig]):
+    """Simple pipeline builder.
+
+    This pipeline will simply pass the user query to the response synthesizer.
+    There are no prompt templates used in this pipeline.
+
+    Inherits attributes from `PipelineBuilderPlugin`.
+    """
+
+    name = "simple"
```

Comment:
```suggestion
    name = "simple_pipeline"
```
minta tolong diupdate sesuai module name / foldernya

Reply:
udpated kak